extensions/hibernate-reactive/runtime/pom.xml

```diff
@@ -34,6 +34,17 @@
         <dependency>
             <groupId>org.hibernate.reactive</groupId>
             <artifactId>hibernate-reactive-core</artifactId>
+            <exclusions>
+                <!-- Make sure Quarkus can manage Mutiny via quarkus-mutiny -->
+                <exclusion>
+                    <groupId>io.smallrye.reactive</groupId>
+                    <artifactId>mutiny</artifactId>
```

Comment:
Is this really necessary? I'd expect the dependency management in the BOM to be enough.

Reply:
Yeah, the exclusion is not needed but doesn't hurt either.